frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -1,58 +1,95 @@
 import React, { useState } from "react";
-import { Box, Button } from "@chakra-ui/react";
+import { Box } from "@chakra-ui/react";
 import CampDetails from "./CampDetails";
 import ScheduleSessions from "./ScheduleSessions";
 import RegistrationForm from "./RegistrationForm";
+import Stepper, { CampCreationPages } from "./StepperNavigation/index";
 
-enum CampCreationPages {
-  CampCreationDetailsPage,
-  ScheduleSessionsPage,
-  RegistrationFormPage,
-}
+const CampCreationPage = (): React.ReactElement => {
```

Comment:
I think we'll (unfortunately) need to have all response state in this main component, since having it in the subpage components themselves don't persist the state when we navigate between pages.